tdp/cli/commands/browse.py

```diff
@@ -134,16 +134,16 @@ def print_formatted_deployment(deployment_log):
 
 def print_formatted_operation_log(operation_log):
     headers = OperationLog.__table__.columns.keys()
-    service_headers = ServiceComponentLog.__table__.columns.keys()
+    service_headers = ComponentVersionLog.__table__.columns.keys()
     # TODO: this outputs Service and ServiceComponent version when it should
```

Comment:
This comment use "ServiceComponent" which should be updated no ?

Reply:
Done